app/src/main/java/com/aleksandra/go4mytrip/ui/main/clothes/PackingClothesFragment.kt

```diff
@@ -0,0 +1,60 @@
+package com.aleksandra.go4mytrip.ui.main.clothes
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.aleksandra.go4mytrip.PackingListRepository
+import com.aleksandra.go4mytrip.R
+import com.aleksandra.go4mytrip.lists.PackingListAdapter
+import com.aleksandra.go4mytrip.lists.PackingModel
+import com.aleksandra.go4mytrip.trips.TripModel
+import com.google.firebase.database.DatabaseReference
+
+class PackingClothesFragment : Fragment(), PackingClothesContract.View {
+
+    private lateinit var recyclerView: RecyclerView
+    private lateinit var referencePackingList: DatabaseReference
+    private lateinit var packingListRepository : PackingListRepository
+    private lateinit var uid: String
+
+    val presenter by lazy { PackingClothesPresenter(this) }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        packingListRepository = PackingListRepository()
+        referencePackingList = packingListRepository.getDatabaseReferencePackingList()
+        uid = packingListRepository.getUid()
+    }
+
+    companion object {
+        private lateinit var idTrip: String
```

Comment:
trip model should be passed to the fragment as its argument, fragment should pass it to the presenter, presenter should store and use it